src/index.js

```diff
@@ -117,7 +117,7 @@ function App() {
           <Route path="/implicit/callback" component={LoginCallback} />
           <Route path="/instructor" component={InstructorHome} />
           <Route path="/parent" component={ParentDashboard} />
-          <Route path="/parent2" component={ParentHome} />
+          <Route path="/parent2" component={ParentDashboard} />
```

Comment:
I think this can be deleted, as there is no parent2 route/component

Reply:
This is deleted!